workshops/forms.py

```diff
@@ -412,3 +413,18 @@ class Meta:
         model = Task
         fields = '__all__'
         widgets = {'person': HiddenInput}
+
+
+class HostForm(forms.ModelForm):
+    domain = forms.CharField(
+        max_length=Host._meta.get_field('domain').max_length,
+        validators=[
+            RegexValidator('[^\w\.-]+', inverse_match=True,
+                           message='Do NOT enter "http://" part or the '
```

Comment:
Change the message to "Please enter only the domain (such as math.esu.edu) without a leading http:// or a trailing /"